rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -796,20 +796,20 @@ async fn run_controller(
         win.show().context("Couldn't show Welcome window")?;
     }
 
+    let tokio_handle = tokio::runtime::Handle::try_current()
```

Comment:
```suggestion
    let tokio_handle = tokio::runtime::Handle::current()
```

If you are gonna panic anyway, might as well use `current`.